tactile_calibration/tactile_state_calibrator/src/tactile_state_calibrator.cpp

```diff
@@ -85,9 +86,9 @@ bool TactileStateCalibrator::extract_idx_range(const YAML::Node &node, std::map<
             {
               ROS_DEBUG_STREAM("  index range valid" );
               // add the range to the idx_map and associate it to the calibration pointer
-              for (unsigned int i = idx_ranges[0]; i <= idx_ranges[1];++i) 
+              for (int i = idx_ranges[0]; i <= idx_ranges[1]; ++i)
```

Comment:
If I am not mistaken i  can never be negative in this filling up case, so why lose half of the potential range of the int ?

Reply:
Change to unsigned int everywhere.